CSS/Styles.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable no-descending-specificity */
 *,
```

Comment:
Your design has a margin to both sides that is not present in the template design. Please fix them.

<img width="228" alt="Screen Shot 2022-06-29 at 18 10 31" src="https://user-images.githubusercontent.com/82426682/176560701-2a389679-21eb-4121-972f-c8da050a7434.png">

<img width="484" alt="Screen Shot 2022-06-29 at 18 10 50" src="https://user-images.githubusercontent.com/82426682/176560728-f30009b6-fe81-4581-83b9-9c239b645b03.png">

Reply:
@vechicin Could you please clarify this for me? There's no margin on my design or on the figma template